tokens/connect/polkadotjsextension.md

```diff
@@ -0,0 +1,90 @@
+ 2. Find Moonriver in the list and select any one of the providers in the dropdown
+ 3. Click the "Switch" Button at the Top and wait for the page to reload
+
+![Switch to the correct network](/images/tokens/connect/polkadotjsext/polkadotjs-ext-2.png)
+
+
+Ensure you're on the Moonriver or Moonbeam Network, then take the following steps: 
+
+ 1. Click on the "Settings" Tab
+ 2. Navigate to "Metadata"
+ 3. Click on "Update Metadata". If you see a message that reads `No Upgradable extensions`, you can skip the following steps
+ 4. The Polkadot.js extension will pop up to prompt you to confirm the update. As a safety measure, verify the URL displayed in the extension is the same as the Polkadot.js Apps URL in your browser
+ 5. Click "Yes, do this metadata update" to complete the update  
+
+It's a good idea to repeat the `Update metadata` process for each network that you'd like to interact with. 
+
+![Update Metadata](/images/tokens/connect/polkadotjsext/polkadotjs-ext-3.png)
+
+
+
+## Creating a Moonbeam Address {: #creating-a-moonbeam-address }
+Open the Polkadot.js extension and click on the "+" Icon. Then press "Create a New Account". This will open the “Create an account” wizard. In the first page, the twelve word mnemonic seed is shown. Make sure you backup your twelve word mnemonic seed. The words must be kept in the exact order displayed on your screen. For more information on storing your mnemonic seed safely, see [this link](https://wiki.polkadot.network/docs/learn-account-generation#storing-your-key-safely){target=_blank}. On this page, take the following steps:
+
+ 1. Ensure you have your mnemonic seed securely backed up, then check the corresponding box. If you navigate away from the extension before completing the account setup wizard, the seed will not be saved and you'll need to create a new account.
+ 2. Click on the “next step” button
+
+!!! note
+    Moonbeam uses Ethereum-style H160 addresses across all of its networks - Moonbeam, Moonriver, and Moonbase Alpha. There is no need to create separate accounts for use on each chain.
+
+![New Account](/images/tokens/connect/polkadotjsext/polkadotjs-ext-4.png)
+
+!!! note
+    DO NOT share your seed with anyone. Your seed grants full access to ALL of your funds. 
+
+On this final page of the account creation wizard, take the following steps:
+
+ 1. Select the `Moonriver` Network. If you create an account in another format, such as Polkadot or Kusama, you'll be unable to later convert that to the Ethereum-style address format used in Moonriver and you'll need to create a new Moonriver address 
+ 2. Give this account a descriptive name
+ 3. Set a password, this will be used when signing transactions
+ 4. Confirm the password
+ 5. Click “Add the account with the generated seed” to complete the account generation 
+
+![Finish Account Creation](/images/tokens/connect/polkadotjsext/polkadotjs-ext-5.png)
```

Comment:
This image is too big. In such cases you can use HTML image imports. Check Nifty Wallet for example

```
<img src="/images/tokens/connect/nifty/nifty-1.png" alt="Set wallet password" style="width: 50%; display: block; margin-left: auto; margin-right: auto;" />
```

Reply:
Thank you for the feedback @albertov19, should be all set now 👍